app/javascript/dashboard/components/layout/sidebarComponents/SecondaryNavItem.vue

```diff
@@ -361,4 +461,36 @@ export default {
     margin-left: var(--space-small);
   }
 }
+
+.dropdown-submenu--wrap {
+  margin-bottom: var(--space-smaller);
+  margin-left: var(--space-two) !important;
+
+  .dropdown-submenu--item {
+    color: var(--s-600);
+    height: var(--space-medium);
+    padding: var(--space-smaller);
+    margin: var(--space-small) 0 0 0;
+
+    &:hover {
+      background: var(--s-25);
+    }
+    &.is-active {
+      background: var(--s-50);
```

Comment:
```suggestion
      background: var(--w-25);
```
Can we try primary color for active items and see if they are good?